CMakeLists.txt

```diff
@@ -42,6 +45,65 @@ if(NOT CMAKE_BUILD_TYPE)
         FORCE)
 endif()
 
+include(cmake/SetBuildParallelLevel.cmake)
+SetBuildParallelLevel(CMAKE_BUILD_PARALLEL_LEVEL)
+
+set(CTEST_OUTPUT_ON_FAILURE TRUE)
+set(CTEST_PARALLEL_LEVEL ${CMAKE_BUILD_PARALLEL_LEVEL})
```

Comment:
I suppose that this line is excess since we set `CTEST_PARALLEL_LEVEL` in the line below.

Reply:
Seems so and moreover this option doesn't work in CMake 2.8, only in CMake 3.10+.
(According to a Git history [^1] the option `CTEST_PARALLEL_LEVEL` was added approx. in 2017 (version ~3.10), when the current required version in Tarantool is 2.8.)

Fixed (`CTEST_PARALLEL_LEVEL` is set as an intermediate var for calculating a final number of jobs):

```diff
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -50,8 +50,8 @@ include(cmake/SetBuildParallelLevel.cmake)
 SetBuildParallelLevel(CMAKE_BUILD_PARALLEL_LEVEL)
 
 set(CTEST_OUTPUT_ON_FAILURE TRUE)
-set(CTEST_PARALLEL_LEVEL ${CMAKE_BUILD_PARALLEL_LEVEL})
-math(EXPR CTEST_PARALLEL_LEVEL "${CMAKE_BUILD_PARALLEL_LEVEL} * 2" OUTPUT_FORMAT DECIMAL)
+math(EXPR CTEST_PARALLEL_LEVEL
+     "${CMAKE_BUILD_PARALLEL_LEVEL} * 2" OUTPUT_FORMAT DECIMAL)
 # TARANTOOL_CTEST_FLAGS is used by CMake targets.
 # XXX: CMake 3.17 introduces CMAKE_CTEST_ARGUMENTS that contains
 # CTest options and is used by `test` target.
```

[^1]: https://gitlab.kitware.com/cmake/cmake/-/commit/e6b77c5f1a918b3b33457eca0590d0ff3840c08f